src/main/java/teammates/ui/template/InstructorFeedbackResultsParticipantPanel.java

```diff
@@ -0,0 +1,87 @@
+package teammates.ui.template;
+
+import teammates.common.util.Url;
+
+
+
+
+public class InstructorFeedbackResultsParticipantPanel implements Comparable<InstructorFeedbackResultsParticipantPanel>{
+    protected boolean isGiver; // if false, then participant is a recipient
+    protected String participantIdentifier; // email, team name, or %GENERAL%
+    protected String name; 
+    
+    protected boolean isHasResponses;
+    protected boolean isEmailValid;
+    
+    protected String profilePictureLink;
+    
+    protected InstructorFeedbackResultsModerationButton moderationButton;
+    
+    public String getParticipantIdentifier() {
+        return participantIdentifier;
+    }
+    
+    public void setParticipantIdentifier(String participantIdentifier) {
+        this.participantIdentifier = participantIdentifier;
+    }
+
+    public boolean isGiver() {
+        return isGiver;
+    }
+
+    public void setGiver(boolean isGiver) {
+        this.isGiver = isGiver;
+    }
```

Comment:
`setIsGiver()`? `setGiver` sounds like it would take in a `String` or something similar.